rustls/src/lib.rs

```diff
@@ -266,6 +266,10 @@
 //!
 //! [`mio`]: https://docs.rs/mio/latest/mio/
 //!
+//! # Detailed manual
+//!
+//! The [`detailed manual`](crate::manual) explains some design decisions and includes some how-to guidance.
```

Comment:
I'd avoid the "some" usage here, it sounds vague.

```suggestion
//! The [`detailed manual`](crate::manual) explains design decisions and includes how-to guidance.
```